docs/contributing.rst

```diff
@@ -30,6 +30,18 @@ Django-crispy-forms comes with git hook scripts. These can be installed by runni
     
 Pre-commit will now run automatically on git-commit and check adherence to the style guide (black, isort & flake8).
 
+Build the documentation locally
+===============================
+
+If you make documentation changes they can be seen locally.
+In the directory where you cloned ``django-crispy-forms``::
+
+    cd docs
+    pip install -r requirements.txt
+    make html
```

Comment:
Or `make.bat html` on windows. It took me a good few weeks to figure out how to install `make` on windows when I was first learning. Could just add as a separate line with a comment for the os is applies to? 

See here for reference. 

https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/

Reply:
Actually, thanks to `make.bat` existing in the `docs`: I think that the current commands would also work on Windows. I guess that Django specifies for clarity though. I will check on a VirtualBox out of curiosity.

I'm happy to add a line to avoid confusion :-)

Reply:
If it works then I'm not sure we need to be that explicit?

Reply:
Confirmed: on Windows it works following the instructions as they are now. Perhaps as it is now is good to go then?

Reply:
I agree.